deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -79,12 +82,23 @@ spec:
     # Default MTU to use for the host gateway interface and the network interface of each Pod.
     # If omitted, antrea-agent will discover the MTU of the Node's primary interface and
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
-    #defaultMTU: 1450
+    #defaultMTU: 0
```

Comment:
Can you confirm in Antrea what's the default value if this is set 0 in ConfigMap, we need to update network CR status for MTU

Reply:
> Can you confirm in Antrea what's the default value if this is set 0 in ConfigMap, we need to update network CR status for MTU

There will be a new patch to find a way to get MTU and update MTU in network status CR.